src/Components/Checkout.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+
+export class Checkout extends Component {
+    constructor(){
+        super()
+
+        this.state = {
+            firstName: "",
+            lastName: "",
+            email: "",
+            credit: "",
+            zip: ""
+        }
+    }
+
+    handleChange = (e) => {
+
+        this.setState({
+            [e.target.name]: e.target.value
+        })
+    }
+
+    handleSubmit = (e) => {
+        e.preventDefault()
+
+        if (this.state.firstName.length === 0 || this.state.lastName.length === 0 || this.state.email.length === 0 || this.state.credit.length === 0 || this.state.zip.length === 0) {
+            alert(`Input is not valid`)
+        } else if (this.state.credit.length < 16) {
+            alert(`Credit card number is not valid`)
+        } else if (this.state.zip.length < 5){
+            alert(`Zip code is not valid`)
+        } else {
+            alert(`Purchase complete`)
```

Comment:
See my comment about passing total price down to Checkout as a prop. You could use that value here in your alert.